### PR TITLE
added optional flag for opentelemetry

### PR DESCRIPTION
**Pull request by abilpraju-aot**
# Issue Tracking
https://aottech.atlassian.net/browse/FWF-1905
JIRA: 
Issue Type: FEATURE
Added opentelemetry as optional flag

### CODE REVIEW

File: forms-flow-web/docker-compose.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ services:
         - REACT_APP_EXPORT_PDF_ENABLED=${EXPORT_PDF_ENABLED:-false}
         - REACT_APP_PUBLIC_WORKFLOW_ENABLED=${PUBLIC_WORKFLOW_ENABLED:-false}
         - REACT_APP_DOCUMENT_SERVICE_URL=${DOCUMENT_SERVICE_URL}
+        - REACT_APP_OPENTELEMETRY_SERVICE=${OPENTELEMETRY_SERVICE}
```
Comment (sreehari-aot) [inline forms-flow-web/docker-compose.yml:32]: can add default to false

File: deployment/docker/docker-compose.yml [hunk 2]
```diff
@@ -152,6 +152,7 @@ services:
         - REACT_APP_EXPORT_PDF_ENABLED=${EXPORT_PDF_ENABLED:-false}
         - REACT_APP_PUBLIC_WORKFLOW_ENABLED=${PUBLIC_WORKFLOW_ENABLED:-false}
         - REACT_APP_DOCUMENT_SERVICE_URL=${DOCUMENT_SERVICE_URL}
+        - REACT_APP_OPENTELEMETRY_SERVICE=${OPENTELEMETRY_SERVICE}
```
Comment (sreehari-aot) [inline deployment/docker/docker-compose.yml:155]: better to give default value

File: deployment/docker/README.md [hunk 3]
```diff
@@ -156,6 +156,7 @@ Variable name | Meaning | Possible values | Default value |
 `EXPORT_PDF_ENABLED`|Manage export to pdf feature|true/false
 `PUBLIC_WORKFLOW_ENABLED`|Enable creating workflow for all tenants
 `DOCUMENT_SERVICE_URL`|Formsflow document service api url||`http://{your-ip-address}:{port}`
+`OPENTELEMETRY_SERVICE`|Formsflow opentelemetry service url||`false`
```
Comment (sreehari-aot) [inline deployment/docker/README.md:159]: is it URL or boolean ?

**Review by sreehari-aot — APPROVED** [review]
LGTM

**Review by abhilash-aot — APPROVED** [review]

**Pull request merged**